<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="SimulatorView" name="simulatorView"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="plusDist">
        <property name="text">
         <string>+50cm</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="minusDist">
        <property name="text">
         <string>-50cm</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pauseButton">
        <property name="text">
         <string>Pause</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="plusRot">
        <property name="text">
         <string>+60°</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="minusRot">
        <property name="text">
         <string>-60°</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QCustomPlot" name="plot" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>200</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SimulatorView</class>
   <extends>QGraphicsView</extends>
   <header>SimulatorView.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pauseButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>pause()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>413</x>
     <y>130</y>
    </hint>
    <hint type="destinationlabel">
     <x>235</x>
     <y>233</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>minusDist</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>minusDist()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>167</x>
     <y>300</y>
    </hint>
    <hint type="destinationlabel">
     <x>274</x>
     <y>264</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>minusRot</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>minusRot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>489</x>
     <y>300</y>
    </hint>
    <hint type="destinationlabel">
     <x>274</x>
     <y>264</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>plusDist</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>plusDist()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>60</x>
     <y>300</y>
    </hint>
    <hint type="destinationlabel">
     <x>274</x>
     <y>264</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>plusRot</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>plusRot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>382</x>
     <y>300</y>
    </hint>
    <hint type="destinationlabel">
     <x>274</x>
     <y>264</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>pause()</slot>
  <slot>plusDist()</slot>
  <slot>plusRot()</slot>
  <slot>minusDist()</slot>
  <slot>minusRot()</slot>
 </slots>
</ui>
